package.json

```diff
@@ -142,7 +150,7 @@
         }
       ],
       "strict": [
-        "error",
+        "off",
```

Comment:
Side note: This rule is good, but is very noisy since modules use 'use strict' by default. This will need to be followed up on, since deleting all of the 'use strict' lines at the top of each file:
1. Didn't work very well with `--fix`
1. Makes the diff really difficult to read